getting_started/index.md

```diff
@@ -1,4 +1,4 @@
-# Getting started with Mbed TLS
+# Getting Started
```

Comment:
Like Tom, I don't think having “Mbed TLS” in the title is particularly jarring.

Also, knowledge base article titles currently consistently use normal case, not title case.

Reply:
Re title case, this is true of many knowledge base articles, but not of section headings (of which "Getting Started" is now one). Sections are usually title case - for reference, see the section headings: ["Knowledge Base"](https://mbed-tls.readthedocs.io/en/latest/kb/) and ["Compiling and Building"](https://mbed-tls.readthedocs.io/en/latest/kb/compiling-and-building/).

Reply:
Looking further, the “Getting started” page is actually a table of contents page, and there are separate getting-started guides for downloading, for building, and for the PSA API. So I now agree that it makes sense to rename the page to be just “Getting started”.

Regarding title case: I personally dislike title case and think that in the 21st century Internet world it feels affected. But indeed it's currently commonly used for pages that represent sections on the website. In the [old site](https://web.archive.org/web/20190107111130/https://tls.mbed.org/), this was inconsistent (e.g. “Knowledge Base” and “Source Code” but “Dev corner” and “API reference”). Anyway it's a cosmetic thing so I wouldn't reject on that basis.

Reply:
In that case I'll revert the more controversial "Long-term plans" change to its original and just stick with the "Getting Started" rename.